pkg/arvo/neo/cod/std/src/imp/accel-conf.hoon

```diff
@@ -0,0 +1,97 @@
+/@  accel-conf
+=> 
+|%
+++  card  card:neo
+++  build
+  |=  [=bowl:neo conf=accel-conf]
+  ^-  (quip card pail:neo)
+  :_  accel-conf/!>(conf)
+  =/  =term
+    `@tas`(cat 3 'accel-' (scot %t (spat (pout (snip here.bowl)))))
+  =/  =pith:neo
+    #/[p/our.bowl]/cod/std/out/imp/[term]
+  =;  =vase
+    [pith %make %ford-riff `vase/vase ~]^~
```

Comment:
Are the changes to `app/neo` to enable this line? "If the head of the "pail" is a vase rather than a stud, use the contents of that vase rather than looking for an imp file."